terraform/output.tf

```diff
@@ -0,0 +1,5 @@
+output "flag_in_var_tmp" {
+  value = "${random_string.flag_in_var_tmp.result}"
+  sensitive = true
```

Comment:
Hmm, I wonder how the user or application running the tf module would get hold of the flag and if it should be a variable instead (i.e. provided from outside)? But we can change that later.

Reply:
I don't know, but yes we can change that later :)